recipe/meta.yaml

```diff
@@ -50,6 +58,8 @@ test:
     - r-testthat
     # NB: r-webutils creates a circular dependency; it should be temporarily disabled during migrations
     - r-webutils
+    # TODO: See above. Remove explicit dependency when is workaround not needed anymore.
+    - openssl <3.2.0a0
```

Comment:
The constraint here is just so that it picks a version for which the test works.
Normally, the user will get the latest `libcurl=8`/`openssl=3` versions.